.aegir.js

```diff
@@ -8,7 +8,7 @@ const ipfsdServer = IPFSFactory.createServer()
 const preloadNode = MockPreloadNode.createNode()
 
 module.exports = {
-  bundlesize: { maxSize: '689kB' },
+  bundlesize: { maxSize: '756KB' },
```

Comment:
I'm not sure this is change is needed, In this branch I'm getting the following:

```
⨎ npx aegir build --bundlesize
Child
       1466 modules
Child
       1466 modules
 PASS  ./dist/index.min.js: 682.3KB < maxSize 756KB (gzip) 
```

Reply:
Will revert since we're disabling it anyway